stackslib/src/net/codec.rs

```diff
@@ -859,70 +918,66 @@ impl StacksMessageCodec for RelayData {
 
 impl StacksMessageType {
     pub fn get_message_id(&self) -> StacksMessageID {
-        match *self {
-            StacksMessageType::Handshake(ref _m) => StacksMessageID::Handshake,
-            StacksMessageType::HandshakeAccept(ref _m) => StacksMessageID::HandshakeAccept,
+        match self {
```

Comment:
Can you please keep refactoring separate from the new features?  This is only going to increase merge conflicts for my work with the new inventory state machine.